src/elements/index.js

```diff
@@ -24,6 +24,8 @@ export { default as Input } from './Input/Input'
 export { default as Label } from './Label/Label'
 export { default as LabelDetail } from './Label/LabelDetail'
 
+// TODO: Export `List`
```

Comment:
Looks like this is done now, but we should export the rest of the List sub components as well.  I'll add a common test that ensures all components are exported at the top level as well.